src/main/java/com/develhope/spring/vehicles/controller/VehicleController.java

```diff
@@ -37,6 +36,19 @@ public ResponseEntity<List<Vehicle>> getFilteredVehicles(
                 .body(vehicleServiceFilter.getFilteredVehicles());
     }
 
+    @GetMapping("/mostExpensiveSoldedVehicle")
```

Comment:
dato che il filtro fa riferimento a soldedVehicle, mi chiedo se si debba prendere in considerazione Vehicle o Order(che interpretiamo infatti come veicoli venduti)